kamon-statsd/src/main/scala/kamon/statsd/StatsDMetricsSender.scala

```diff
@@ -48,7 +48,7 @@ class StatsDMetricsSender(remote: InetSocketAddress, maxPacketSizeInBytes: Long,
   }
 
   def writeMetricsToRemote(tick: TickMetricSnapshot, udpSender: ActorRef): Unit = {
-    val packetBuilder = new MetricDataPacketBuilder(maxPacketSizeInBytes, udpSender, remote)
+    val packetBuilder = new MetricDataPacketBuilder(maxPacketSizeInBytes, udpSender, statsDHost, statsDPort)
```

Comment:
That's right, I will do that. To be honest, I thought about some sort of InetSocketAddresProvider instance that will allow you to either create one an instance on demand or a single one depending on how you guys wanted to go in the future, but honestly I don't think there's a need to just create one instance, it's cheap to create them always on-the fly since they get discarded right away.